code/modules/bitrunning/components/digital_aura.dm

```diff
@@ -0,0 +1,45 @@
+/// Creates a digital effect around the target
+/datum/element/digital_aura
```

Comment:
This should be just a generic effect in animations.dm instead of an element. You don't do anything particularly complex worth isolating, or anything that requires the effect be a separate object.